packages/airnode-utilities/src/logging/logging.ts

```diff
@@ -1,62 +1,68 @@
-import { LogLevel, LogOptions, PendingLog, LogMetadata, LogConfig } from './types';
+import { LogLevel, LogOptions, ErrorLogOptions, PendingLog } from './types';
 import { formatDateTimeMs } from '../date';
 
+let logOptions: LogOptions;
+
+export const getLogOptions = () => {
+  return logOptions;
+};
+
+export const setLogOptions = (newLogOptions: LogOptions) => {
+  logOptions = newLogOptions;
+};
+
 const logLevels: { readonly [key in LogLevel]: number } = {
   DEBUG: 0,
   INFO: 1,
   WARN: 2,
   ERROR: 3,
 };
 
-export function buildBaseOptions(config: LogConfig, meta: LogMetadata) {
-  return {
-    format: config.nodeSettings.logFormat,
-    level: config.nodeSettings.logLevel,
-    meta,
-  };
-}
-
 export const logger = {
-  log: (message: string, options?: LogOptions) => {
+  options: 'test',
```

Comment:
What is this field for ? Why is it called `test`?

Reply:
Thanks for catching, that should not be there, removed.

I've also tested these already with an AWS deployment so I'll go ahead and merge.